main/guides/js-programming/far.md

```diff
@@ -69,11 +67,11 @@ You should call `Far()` on an object if it both:
     you should run `Far()` on it after creating it.
 - Has methods called on it, as opposed to just effectively storing data.
 
-As defined, only passables can be either passed as arguments or returned as results,
+Only passables can be passed as arguments or returned as results,
 and they must be hardened. If the passable is a remotable, it must be hardened with `Far()`.
 Otherwise, it must be hardened with [`harden()`](./ses/ses-guide.md#harden).
 
-There's no harm in using `Far()` on an object, even if it never leaves its vat. An error
+An error
 is thrown if you call `Far()` on a record, instead of an object, which doesn't have function
 values. However, if object `foo` should never be exposed to other vats, you should make it
 a point **not** to use `Far()` on it. If `foo` is not marked as `Remotable` but is accidentally
```

Comment:
The term "remotable" here is a category for talking about objects. It is not itself an identifier that would appear in code.

```suggestion
a point **not** to use `Far()` on it. If `foo` is not marked as a remotable but is accidentally
```